### DISCUSSION TITLE
404 error code

### DISCUSSION THREAD

**Discussion by Secker17** [community/community#35267] [Other Feature Feedback, Questions, & Ideas]
How do i fix this?
![Skjermbilde 2022-10-06 013223](https://user-images.githubusercontent.com/96775895/194182214-778ca527-e497-4e34-97e1-4e8ca9231034.png)

**Comment by radar-blip** [comment]
You need index.html in your repository (and .css that goes along with it).
Or a folder that contains your index.html with link pointing to that folder + index.html

**Comment by yoannchaudet** [comment]
Your Pages site is published at https://secker17.github.io/NSMP/.

If you want it one level up, i.e. at `secker17.github.io`, rename your `NSMP` repository to `secker17.github.io`.

**Reply by 3kh0** [reply]
@Secker17 Please mark this as an answer or whichever one helped you! It helps others find a solution. ❤️
